### ISSUE TITLE
tf.keras.losses.MeanSquaredError doesnt support scalar inputs

### ISSUE THREAD

**Issue by breadbread1984** [tensorflow/tensorflow#36048]
I am using tensorflow version 2.2.0-dev20200119. tf.keras.losses.MeanSquaredError can't handle input scalar values properly. The following code can reproduce the problem.

```python
#!/usr/bin/python3
import tensorflow as tf;
tf.keras.losses.MeanSquaredError()(1,0);
```

**Comment by amahendrakar** [comment]
Was able to reproduce the issue with TF 2.1.0 and 2.2.0-dev20200119. Please find the Gist [here](https://colab.sandbox.google.com/gist/amahendrakar/02862816fff21cc0cb9737852b051ac6/36048.ipynb). Thanks!

**Comment by pavithrasv** [comment]
Hi @breadbread1984, the labels and predictions are expected to be at least 2D for the metric: https://www.tensorflow.org/api_docs/python/tf/keras/metrics/MeanSquaredError?version=nightly#update_state

y_true: Ground truth values. shape = [batch_size, d0, .. dN].
y_pred: The predicted values. shape = [batch_size, d0, .. dN].
